tests/support/server.tcl

```diff
@@ -240,6 +256,8 @@ proc create_server_config_file {filename config} {
 proc spawn_server {config_file stdout stderr} {
     if {$::valgrind} {
         set pid [exec valgrind --track-origins=yes --trace-children=yes --suppressions=[pwd]/src/valgrind.sup --show-reachable=no --show-possibly-lost=no --leak-check=full src/redis-server $config_file >> $stdout 2>> $stderr &]
+    } elseif ($::sanitizer) {
+        set pid [exec /usr/bin/env ASAN_OPTIONS=allocator_may_return_null=1 src/redis-server $config_file >> $stdout 2>> $stderr &]
```

Comment:
i think maybe we can just apply this change on the default execution (the last `else` in this chain),
but if not, please be aware that instances.tcl (cluster and sentinel tests) spawns severs on it's own (not using this code), see `exec_instance`

Reply:
This is only required for corrupt_dump tests, so I added here only. Anyway, I'll set it by default.

Reply:
ok, make it by default and add a comment stating what it's for...

Reply:
done